src/test/efsw-test.cpp

```diff
@@ -33,7 +33,7 @@ class UpdateListener : public efsw::FileWatchListener {
 
 	void handleFileAction( efsw::WatchID watchid, const std::string& dir,
 						   const std::string& filename, efsw::Action action,
-						   std::string oldFilename = "" ) {
+						   std::string oldFilename = "" ) override {

```

Comment:
Override is giving you warnings in what escenario? If you don't use it should not trigger warnings by default.

Reply:
Keep it, no problem, it's the "modern" way.